### PR TITLE
sysadmin can share repo with admin permission

### PR DESCRIPTION
**Pull request by zMingGit**

### CODE REVIEW

File: seahub/api2/endpoints/admin/shares.py [hunk 1]
```diff
@@ -298,22 +335,24 @@ def put(self, request, repo_id, path, share_type):
                 error_msg = 'User %s not found.' % email
                 return api_error(status.HTTP_404_NOT_FOUND, error_msg)
 
+            if not has_shared_to_user(repo.repo_id, path, email):
+                error_msg = 'Can not find shared record about %s user' % email
```
Comment (xiez) [inline seahub/api2/endpoints/admin/shares.py]: 用标准的404 error msg

File: seahub/api2/endpoints/admin/shares.py [hunk 2]
```diff
@@ -328,26 +367,28 @@ def put(self, request, repo_id, path, share_type):
                 error_msg = 'Group %s not found' % group_id
                 return api_error(status.HTTP_404_NOT_FOUND, error_msg)
 
+            if not has_shared_to_group(repo.repo_id, path, group_id):
+                error_msg = 'Can not find shared record about %s group' %  group.group_name
```
Comment (xiez) [inline seahub/api2/endpoints/admin/shares.py]: 同上

File: seahub/api2/endpoints/admin/shares.py (hunk 1 above)
Comment (zMingGit) [reply seahub/api2/endpoints/admin/shares.py]: 是指xxx not found 吗.

File: seahub/templates/js/sysadmin-templates.html [hunk 3]
```diff
@@ -709,21 +709,44 @@ <h3><%= title %></h3>
...
+            {% trans "Admin" %}
         <% } else { %>
-            {% trans "Read-Only" %}
+            <% if (permission == 'rw') { %>
+                {% trans "Read-Write" %}
+            <% } else { %>
+                {% trans "Read-Only" %}
+            <% } %>
         <% } %>
+
```
Comment (llj) [inline seahub/templates/js/sysadmin-templates.html]: remove the empty line.

**Pull request merged**